classes/lang/KeysReference/MetaLang.php

```diff
@@ -56,26 +56,26 @@
 
 trans('Shop powered by PrestaShop', 'Shop.Navigation');
 
-trans('Brand list', 'Shop.Navigation');
+trans('Brands list', 'Shop.Navigation');
 
 trans('Our new products', 'Shop.Navigation');
 
 trans('Enter the e-mail address you use to sign in to receive an e-mail with a new password', 'Shop.Navigation');
 
-trans('On-sale products', 'Shop.Navigation');
+trans('Our special products', 'Shop.Navigation');
 
 trans('Lost ? Find what your are looking for', 'Shop.Navigation');
 
 trans('Suppliers list', 'Shop.Navigation');
 trans('page-not-found', 'Shop.Navigation');
 trans('best-sales', 'Shop.Navigation');
 trans('contact-us', 'Shop.Navigation');
-trans('manufacturers', 'Shop.Navigation');
+trans('brands', 'Shop.Navigation');
```

Comment:
```suggestion
trans('brands', 'Shop.Theme.Catalog');
```
The string already exists in Shop.Theme.Catalog